CI_CD/scripts/model_promotion.py

```diff
@@ -0,0 +1,89 @@
+from model import *
+from data import get_dataloader
+import neptune.new as neptune
+import os
+import subprocess
+import sys
+
+project_name = 'common/pytorch-integration'
+
+# Fetch project 
+project = neptune.get_project(
+    name = project_name,
+    api_token=os.getenv("NEPTUNE_API_TOKEN")
+)
+
+# In-prod
+# Filter in-prod run
+prod_runs_table_df = project.fetch_runs_table(tag='in-prod').to_pandas()
+prod_run_id = prod_runs_table_df['sys/id'].values[0]
+
+# Retrieve run in read mode
+prod_run = neptune.init(
+    project=project_name,
+    api_token=os.getenv("NEPTUNE_API_TOKEN"),
+    run = prod_run_id,
+    mode = 'read-only'
+)
+
+# Stagging
+# filter stagging run
+stagging_runs_table_df = project.fetch_runs_table(tag='stagging').to_pandas()
+stagging_run_id = stagging_runs_table_df['sys/id'].values[0]
+
+# Retrieve run
+stagging_run = neptune.init(
+    project=project_name,
+    api_token=os.getenv("NEPTUNE_API_TOKEN"),
+    run = stagging_run_id,
+    mode = 'read-only'
+)
+
+# Fetching and downloading data from Neptune
+parameters = prod_run['config/hyperparameters'].fetch()
```

Comment:
I think we should have it wrapped in a function called build_model from run metadata or something like that to make it obvious what is happening here. 

Cause, right now it seems that I am using prod parameters to load the stagging model and I am doing the same thing for both models (as I am getting them).